ingestion/src/metadata/ingestion/source/database/databricks/usage.py

```diff
@@ -102,6 +102,9 @@ def process_table_query(self) -> Optional[Iterable[TableQuery]]:
                                 endTime=row.get("execution_end_time_ms"),
                                 analysisDate=datetime.now(),
                                 serviceName=self.config.serviceName,
+                                duration=row.get("duration") / 1000
+                                if row.get("duration")
+                                else None,
```

Comment:
```suggestion
                                duration=row.get("duration", 0) / 1000
                                or None,
```

Reply:
actually I made do it this way voluntarily because it's possible for `row.get("duration")` to be 0 and `0 or None` will return None so could be a misinformation similarly if we just do `row.get("duration",0)/1000` then if duration is not available then by default it will show 0 as duration which again could be a misinformation.


WDYT?